### PR TITLE
Improvements for terraform checks

### PR DESCRIPTION
**Pull request by alexburlton-sonocent**
Trello: https://trello.com/c/DhXg6v9M/183-terraform-daktari-improvements

In this PR:

 - Add basic file existence check (so we can check gcloud auth has been done)
 - Pull out separate check for `tfenv` being installed, and patch up existing terraform check so it gives correct advice if using this (should just be `tfenv install`, not install by hand)
 - Improve helm repo checks to be less brittle and give more useful output

### CODE REVIEW

File: daktari/checks/terraform.py [hunk 1]
```diff
@@ -7,26 +7,40 @@
 from daktari.os import OS
 
 
+class TfenvInstalled(Check):
+    name = "tfenv.installed"
+
+    suggestions = {
+        OS.OS_X: "<cmd>brew install tfenv</cmd>",
+        OS.UBUNTU: """<cmd>git clone https://github.com/tfutils/tfenv.git ~/.tfenv</cmd>
+                    <cmd>ln -s ~/.tfenv/bin/* /usr/local/bin</cmd>""",
```
Comment (mattrussell-sonocent) [inline daktari/checks/terraform.py:16]: Do we need a couple extra spaces here for it to line up?

Comment (alexburlton-sonocent) [reply daktari/checks/terraform.py:16]: I tested it locally and it looked fine, but I can align it for code neatness if nothing else :grin:

**Review by mattrussell-sonocent — APPROVED** [review]

File: daktari/checks/terraform.py (hunk 1 above)
Comment (alexburlton-sonocent) [reply daktari/checks/terraform.py:16]: image

Comment (mattrussell-sonocent) [reply daktari/checks/terraform.py:16]: Maybe it trims initial spaces, I forget now!

File: daktari/checks/terraform.py [hunk 2]
```diff
@@ -7,26 +7,40 @@
 from daktari.os import OS
 
 
+class TfenvInstalled(Check):
+    name = "tfenv.installed"
+
+    suggestions = {
+        OS.OS_X: "<cmd>brew install tfenv</cmd>",
+        OS.UBUNTU: """<cmd>git clone https://github.com/tfutils/tfenv.git ~/.tfenv</cmd>
+                      <cmd>ln -s ~/.tfenv/bin/* /usr/local/bin</cmd>""",
```
Comment (mattrussell-sonocent) [inline daktari/checks/terraform.py:16]: Won't you need `sudo` for that?

Comment (alexburlton-sonocent) [reply daktari/checks/terraform.py:16]: ah yeah maybe, can't hurt to add it :+1:

**Pull request merged**